### DISCUSSION TITLE
Please add the ✅ (checkmark) emoji to PR comments

### DISCUSSION THREAD

**Discussion by vilbergs** [community/community#35992] [Pull Requests]
Hey!

I really love that the list of possible reactions in PR comments is limited to a certain amount, it makes the comment feature feel simple and clean. I would however like to request that the ✅ (checkmark) be added as that helps a lot to indicate that a comment has been handled in some way, for example if it's a suggestion for the PR as a whole, I tend to edit the comment with a little checkmark that I've handled it.

Having this as part of the reactions would be really nice!

<img width="354" alt="Screenshot 2022-10-13 at 08 53 43" src="https://user-images.githubusercontent.com/24994841/195527121-6cc94efd-b92c-4e12-8aa6-517aade8851f.png">

**Comment by SaroGFX** [comment]
Really miss this to indicate something has been processed. A 👍  just doesn't communicate that

**Comment by benjub** [comment]
Indeed, the "Done" reaction emoji (probably a checkmark) is really missing: its meaning is clear, and this would avoid cluttering threads with comments merely saying "Done".  I agree that a limited set of reaction emojis is neat (see my comment https://github.com/orgs/community/discussions/10240#discussioncomment-10314309).

**Reply by benjub** [reply]
Also requested by #114260.

**Comment by Utopism** [comment]
+1

**Comment by shyam-hoasted** [comment]
<img width="1791" height="204" alt="Screenshot 2026-04-16 at 8 53 19 PM" src="https://github.com/user-attachments/assets/5bc3a072-dbc2-40b9-8215-639bcdbf380a" />

Yes, just one more <li>✅</li> :D
